cmds/tui.c

```diff
@@ -2417,12 +2595,15 @@ static void tui_window_graph_field(void)
 	wrefresh(win);
 	box(win, 0, 0);
 
-	mvwprintw(win, 1, 2, "Customize fields in graph mode");
+	if (mode == GRAPH_MODE)
+		mvwprintw(win, 1, 2, "Customize fields in graph mode");
+	else
+		mvwprintw(win, 1, 2, "Customize fields in report mode");
 
-	for (i = 0; i < ARRAY_SIZE(graph_field_help); i++)
-		mvwprintw(win, i + 3, 2, "%-*.*s", w-3, w-3, graph_field_help[i]);
+	for (i = 0; i < ARRAY_SIZE(field_help); i++)
+		mvwprintw(win, i + 3, 2, "%-*.*s", w-3, w-3, field_help[i]);
 
-	display_graph_field(win, selected_field, graph_field_flags);
+	display_tui_field(win, selected_field, graph_field_flags, report_field_flags, mode);
```

Comment:
It'd be nice if we can pass single field (graph or report) based on the current mode.